css/properties/font-smooth.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "font-smooth": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-smooth",
+          "support": {
+            "chrome": {
+              "version_added": "5"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "25",
+              "notes": "A similar property is implemented in Firefox called <code>-moz-osx-font-smoothing</code>. It only works on Mac OS X/macOS."
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "4",
+              "notes": "A similar property is implemented in WebKit called <code>-webkit-font-smoothing</code>. It only works on Mac OS X/macOS."
+            },
+            "safari_ios": {
+              "version_added": null,
+              "notes": "A similar property is implemented in WebKit called <code>-webkit-font-smoothing</code>. It only works on Mac OS X/macOS."
```

Comment:
ditto

Reply:
I have removed the notes for `safari_ios`, because afaik it doesn't run on Mac OS X/macOS.